src/BloomExe/MiscUI/BloomIntegrityDialog.cs

```diff
@@ -37,6 +37,11 @@ public static bool CheckIntegrity()
 			
 			foreach(var fileName in files)
 			{
```

Comment:
I did things this way because I prefer runtime checks to #if generally speaking.  And the #if approach here (applied to the data initialization) is a bit messy.